scripts/multiqc

```diff
@@ -459,10 +439,17 @@ plots_flat, plots_interactive, make_pdf, config_file, cl_config, verbose, quiet,
                 logger.warning("Deleting    : {}   (-f was specified)".format(os.path.relpath(config.output_fn)))
                 os.remove(config.output_fn)
             else:
-                logger.error("MultiQC Report {} already exists.".format(config.output_fn))
+                report_num = 0
                 logger.info("Use -f or --force to overwrite existing reports")
-                shutil.rmtree(tmp_dir)
-                sys.exit(1)
+                while os.path.exists(output_name):
+                    output_name, file_ending = os.path.splitext(config.output_fn)
+                    output_name.append("_{}".format(report_num))
+                    output_name.append(file_ending)
+                    report_num += 1
+                #We keep going until a new name is found, set the new name as the name to use. 
+                config.output_fn=output_name
```

Comment:
Would be good to add a `logger.warning` message here saying that there is a conflicting filename, report will be saved as _x_...